llvm/utils/lit/tests/Inputs/shtest-cat/cat-newline.txt

```diff
@@ -0,0 +1,12 @@
+# RUN: cat %S/txt-files/newline | FileCheck %s
```

Comment:
Maybe direct this to a file w/ `> %t`, and then you can assert some things about the output? I was thinking verifying the length and other properties, but you could probably just diff it against the original file.